press-this.php

```diff
@@ -983,23 +983,82 @@ public function register_options_page() {
 	 * @see self::register_options_page()
 	 */
 	public function do_options_page() {
+
+	/**
+	 * Adding the following <style> block here. I need to review how to properly add CSS to this page
+	 */		
 		?>
+		<style type="text/css">
+			.postbox-pt {
+				margin: 1em 0 0 0;
+				padding: 0 1em 1em;
+			}
+
+				.postbox-pt textarea {
+					width: 100%;
+					font-size: 1em;
+				}
+
+				.postbox-pt h4 {
+					margin: 2em 0 1em;
+				}
+
+				.postbox-pt-buttons {
+					line-height: 2em;
+				}
+
+				.button-pt-bookmarklet:before {
+					color: #FFF;
+					font: 400 20px/1 dashicons;
+					content: '\f157';
+					position: relative;
+					display: inline-block;
+					top: 2px;
+				}
+
+		</style>
 		<div class="wrap">
-			<h2>Press This</h2>
-			<form>
-			<div class="tool-box">
-				<p><?php _e('Press This is a bookmarklet: a little app that runs in your browser and lets you grab bits of the web.');?></p>
+			<h2><?php echo get_admin_page_title() ?></h2>
+			<div class="postbox postbox-pt">
+				<h3><?php _e('What it is?'); ?></h3>
+				<p><?php _e('Press This is a little app that lets you grab bits of the web and create new posts with ease.');?></p>
 				<p><?php _e('Use Press This to clip text, images and videos from any web page. Then edit and add more straight from Press This before you save or publish it in a post on your site.'); ?></p>
-				<p class="description"><?php _e('Drag-and-drop the following link to your bookmarks bar or right click it and add it to your favorites for a posting shortcut.') ?></p>
-				<p class="pressthis"><a onclick="return false;" oncontextmenu="if(window.navigator.userAgent.indexOf('WebKit')!=-1||window.navigator.userAgent.indexOf('MSIE')!=-1){jQuery('.pressthis-code').show().find('textarea').focus().select();return false;}" href="<?php echo htmlspecialchars( get_shortcut_link() ); ?>"><span><?php _e('Press This') ?></span></a></p>
-				<div class="pressthis-code" style="display:none;">
-					<p class="description"><?php _e('If your bookmarks toolbar is hidden: copy the code below, open your Bookmarks manager, create new bookmark, type Press This into the name field and paste the code into the URL field.') ?></p>
-					<p><textarea rows="5" cols="120" readonly="readonly"><?php echo htmlspecialchars( get_shortcut_link() ); ?></textarea></p>
+			</div>
+			<form>
+			<div class="postbox postbox-pt">
+				<h3><?php _e('How to install it?'); ?></h3>
+				<h4><?php _e('Press This Bookmarklet'); ?></h4>
+				<p><?php _e('This method is faster and almost instantaneous. To use it, drag-and-drop the following link to your bookmarks bar:'); ?></p>
```

Comment:
And at the end: "Some mobile browsers make it impossible to add Javascript bookmarklets. For those, use the direct link version below."